draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -555,6 +555,79 @@ To ensure readability and replicability, the examples in this specification do
 not contain decoy digests unless explicitly stated. For an example
 with decoy digests, see (#example-simple_structured).
 
+### Recursive Disclosures {#recursive_disclosures}
+
+The algorithms above are compatible with "recursive disclosures", in which one
+selectively disclosed field reveals the existence of more selectively
+disclosable fields.  For example, consider the following JSON structure:
+
+```json
+{
+    "family_name", "Möbius",
+    "nationalities": ["DE", "FR", "UK"] 
+}
+```
+
+The issuer may wish to allow the holder to reveal no nationalities at all, but
+require that if any nationalities are disclosed, then "DE" must be disclosed.
```

Comment:
```suggestion
When the Holder has multiple nationalities, the issuer may wish to allow the holder to reveal
each of those nationalities individually.
```

Reply:
The example below should be adapted accordingly.